src/main/java/com/altenheim/kalender/controller/viewController/SearchViewController.java

```diff
@@ -87,34 +204,63 @@ private void updateUserStepView(ActionEvent event)
             incrementor = 1;
         }
         else if (button.equals(btnBack) && userStep == 1)        
-            return;           
+            return;  
+        else if (button.equals(btnNextSuggestion) && userStep == 3)
+        {
+            getNextSuggestion();
+            return;  
+        }       
         
         int currentIndex = userStep - 1;
         int requestedIndex = userStep - 1 + incrementor;
         changeViewState(allSteps[currentIndex], allSteps[requestedIndex], images[currentIndex], images[requestedIndex]); 
         userStep += incrementor; 
         txtHeaderStep.setText(headings[currentIndex]);
-
+    }
+    
+    private int suggestions = 1;
+    private void getNextSuggestion()
+    {
+        var startDateInput = startDate.getValue();
+        var startTimeInput = timeStart.getValue();
+        var currentCheck = LocalDateTime.of(startDateInput, startTimeInput);
+        int duration = (int)sliderDurationMinutes.getValue();
+        var newTime = currentCheck.plusMinutes(suggestions * duration);
+        var entry = dateSuggestionController.getDateSuggestionFromEntryList(currentSuggestions, newTime, duration);
+        suggestions++;        
+        SuggestionsModel.addToList(entry.getStartTime(), entry.getEndTime(), entry.getStartDate());
     }
 
     private void startRequest()
     {
         var userPrefs = entryFactory.createUserEntry(startDate.getValue(),
                 endDate.getValue(), timeStart.getValue(), timeEnd.getValue());
-        int duration = (int)sliderAppointmentDuration.getValue();
-        var openingHours = entryFactory.createOpeningHoursWithLunchBreak();
+        int duration = (int)sliderDurationMinutes.getValue();
+        var openingHours = new HashMap<DayOfWeek, List<SerializableEntry>>();
+        try {
+            openingHours = api.getOpeningHours(dropdownEndAtDest.getEditor().getText());
+        } catch (IOException | InterruptedException e) {
+            // TODO Auto-generated catch block
```

Comment:
Auto generated comments

Reply:
S.o.: Weil ich nicht die Exceptionwrapper durchimportieren wollte und wollte das es läuft. Kommt wieder raus wenn die Exceptions in der API vor Ort behandelt werden.